DUI3-DX/Connectors/ArcGIS/Speckle.Connectors.ArcGIS3/Bindings/Factory.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections.Generic;
+using Speckle.Connectors.DUI.Bindings;
+
+namespace Speckle.Connectors.ArcGIS.Bindings;
+
+public static class Factory
```

Comment:
We don't need this class anymore since we register bindings to container and pass them lazily into panel

Reply:
Done!